packages/ui/src/constants.ts

```diff
@@ -21,48 +22,55 @@ export const PAYMENT_INFO_ACCOUNT = '5CXQZrh1MSgnGGCdJu3tqvRfCv7t5iQXGGV9UKotrbf
 
 export const networkList = {
   kusama: {
+    chainId: 'kusama',
     explorerNetworkName: 'kusama',
     rpcUrl: 'wss://kusama-rpc.polkadot.io',
-    wsGraphqlUrl: 'wss://kusama.multix.chainsafe.io/graphql',
-    httpGraphqlUrl: 'https://kusama.multix.chainsafe.io/graphql',
+    wsGraphqlUrl: 'wss://squid.subsquid.io/multix-test/graphql',
```

Comment:
aren't we suppose to use versioned urls?

Reply:
This is the prod url. If we bump the `squid.yaml` manifest version, and run the `sqd deploy` a new version will be created with the version in the url. This can then be tested thoroughly, and upon clicking "make prod" in the Aquarium UI, the versioned api endpoint is directed to this endpoint, with 0 downtime. I should change the name of the project, in order to have just `multix` though, and not "test" in the name